test/system-test/test_proposals.py

```diff
@@ -0,0 +1,37 @@
+import json
+from endpoints import proposalsGet
+from utils import apply_key_release_policy
+from cose.messages import CoseMessage
+
```

Comment:
Can we have other test cases like:
- Check contents of proposalsGet() not only status code when there is no proposal
- Check behavior when more than 5 (`proposalsToKeep`) proposal are made. (check if it's actually old proposals that are deleted)
- Check the array is sorted as intended
?
-

Reply:
The first case is hard to test because we will have a single proposal from setting up the KMS for testing.

The second one sounds good I'll add it

The third one is somewhat covered by the fact we pluck out proposals at specific indices and check which proposal it is, this would be super flaky if it wasn't sorted, checking the created at header isn't impossible but it would make the test infra more complex since we'd need to parse the cbor

Reply:
https://github.com/microsoft/azure-privacy-sandbox-kms/pull/457/commits/3b2095dfdca7040524e38d06e4c8a2d222d4407f